src/ModelInit.jl

```diff
@@ -278,6 +280,10 @@ function fit_object_psfs!{NumType <: Number}(
     raw_central_psf = blob[b].raw_psf_comp(pixel_loc[1], pixel_loc[2])
     central_psf, central_psf_params =
       PSF.fit_raw_psf_for_celeste(raw_central_psf, psf_optimizer, initial_psf_params)
+
+    # Get all relevant sources *in this image*
+    relevant_sources = get_all_relevant_sources_in_image(mp, target_sources, b)
```

Comment:
This will still fail to consider as relevant sources that are outside the image but which influence tiles on the edge of the image.  The problem is ultimately in get_tiled_image_sources, which only considers a single tiled image.

The easiest fix is to ignore this, but if we're going to do that, we should put comments in the code and open an issue at least.  The next easiest fix would be to enforce not fitting objects near the edge of an image.  Doing it right means building tile_sources from all objects rather than just a single image.